registry/storage/driver/gcs/gcs_test.go

```diff
@@ -42,6 +44,11 @@ func init() {
 		return
 	}
 
+	jsonKey, err := ioutil.ReadFile(credentials)
```

Comment:
The `io/ioutil` package is deprecated.
```suggestion
	jsonKey, err := os.ReadFile(credentials)
```

Reply:
+1